packages/ceramic-core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -176,7 +176,7 @@ class InMemoryAnchorService extends AnchorService {
     const didDoc = await this._ceramic.context.resolver.resolve(kid)
     const jws = [_protected, payload, signature].join(".")
     await didJwt.verifyJWS(jws, didDoc.publicKey)
-    return kid
+    return kid.split('?')[0]
```

Comment:
we might have strings of the following format here:
`did:3:bafoier...?version-id=baiuehr...#Ab3d2`
and 
`did:key:Abm32...#Abm32`
We need to account for both of them. Safest is probably to use a regex similar to the did-resolver: https://github.com/decentralized-identity/did-resolver/blob/master/src/resolver.ts#L121-L123

Reply:
I used a similar regex 👍